scipy/stats/_discrete_distns.py

```diff
@@ -5,7 +5,8 @@
 from functools import partial
 
 from scipy import special
-from scipy.special import entr, logsumexp, betaln, gammaln as gamln, zeta
+from scipy.special import entr, logsumexp, betaln, gammaln as gamln
+from scipy.special._ufuncs import _gen_harmonic
```

Comment:
The rest of the file seems to use imports as `scu.<fn>`.

Reply:
Good point! I'll fix that to be consistent with the other uses.

Reply:
Done.